lightly/cli/_helpers.py

```diff
@@ -51,6 +54,18 @@ def fix_input_path(path):
     return path
 
 
+def fix_hydra_arguments(config: str = 'config'):
```

Comment:
This should have two arguments: `config_path` and `config_name`. Also, I think `adapt_hydra_arguments` would probably be the better name but that's just a small thing.

Reply:
I can expose both as arguments. Could you elaborate on where/ why we use different `config_path` and `config_name`? I quickly searched through the code and haven't seen any other use. 

I went for fix and not adapt or update because that's what it is. It's a temporary fix for version incompatibility. Similar to what we did with other fixes from PL or paths in the codebase. Just wanted to be consistent. At one point we hopefully can get rid of some of the rather manual fixes/ patches :D

Reply:
We don't use separate arguments yet but I would just expose both because bit's much cleaner. The `config_path` refers to the name of the directory where the config with `config_name` is stored so they actually mean different things. In our case it's just `config/config.yml`.